quickstart@0.6.rb

```diff
@@ -0,0 +1,35 @@
+require 'fileutils'
```

Comment:
Why is this file name `@0.6.rb` instead of `@1.0.rb` ?

Reply:
Yes, this filename should be `quickstart@1.0.rb`

Reply:
Sure, updated.